physicsnemo/datapipes/cae/domino_datapipe.py

```diff
@@ -744,7 +751,8 @@ def preprocess_surface(self, data_dict, core_dict, center_of_mass, s_min, s_max)
             if self.config.normalize_coordinates:
                 core_dict["surf_grid"] = normalize(core_dict["surf_grid"], s_max, s_min)
                 surface_coordinates = normalize(surface_coordinates, s_max, s_min)
-                surface_neighbors = normalize(surface_neighbors, s_max, s_min)
+                if self.config.num_surface_neighbors_surface > 1:
+                    surface_neighbors = normalize(surface_neighbors, s_max, s_min)
```

Comment:
Why only normalize in the case of multiple surface neighbors? Given the flag name `normalize_coordinates`, this seems like it could lead to unexpected behavior for end users